values.yaml

```diff
@@ -78,7 +77,7 @@ postgresql:
   image:
```

Comment:
Small suggestion: How about adding postgresql port number to values and  template it? Just to maintain the consistency?

Reply:
Added ability to use another external port.